integration-tests/integration/appointments/addAppointment.spec.js

```diff
@@ -11,7 +11,7 @@ context('A user can add an appointment', () => {
   before(() => {
     cy.clearCookies()
     cy.task('resetAndStubTokenVerification')
-    cy.task('stubSignIn', { username: 'ITAG_USER', caseload: 'WWI' })
+    cy.task('stubSignIn', { username: 'ITAG_USER', caseload: 'MDI' })
```

Comment:
A lot of the changes in this PR are to integration tests where the caseload has been set inconsistently. This previously worked because user caseloads were being stubbed as a side effect